docs/app/Examples/modules/Dropdown/Types/DropdownExampleInlineTwo.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Dropdown, Header, Icon } from 'semantic-ui-react'
+
+/**
+ * Heads up!
+ *
+ * SUI core has these 4 fields on the Dropdown.Item:
+ *
+ * "name"  : "Choice 1", // name displayed in dropdown
+ * "value" : "value1", // selected value
+ * "text"  : "Choice 1" // name displayed after selection (optional)
+ * "disabled"  : false // whether field should be displayed as disabled (optional)
+ *
+ * We don't currently make a distinction between 'text' and 'name', but this
+ * example does in the core docs. It would be a breaking change and so let's
```

Comment:
I see, yep this makes sense.  I'd entertain different prop names for these, but no doubt we should support different menu text from active text.  Thanks for clarifying.